### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by felipemarques**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,10 +43,16 @@ $ eval $(docker-machine env mybox)
 
 Also, you should add a route to access containers inside your VM.
 
+For Linux
 ```
 $ sudo route -n add 172.0.0.0/8 $(docker-machine ip $(docker-machine active))
 ```
 
+For Windows ( requires admin elevation )
```
Comment (iamluc) [inline README.md]: You should create a new section like for `Linux` or `Mac OS` or update the Mac OS section

File: README.md [hunk 2]
```diff
@@ -43,10 +43,16 @@ $ eval $(docker-machine env mybox)
 
 Also, you should add a route to access containers inside your VM.
 
+For Linux
```
Comment (iamluc) [inline README.md]: For Mac OS

File: README.md (hunk 1 above)
Comment (felipemarques) [reply README.md]: ok ... sorry

File: README.md (hunk 2 above)
Comment (felipemarques) [reply README.md]: ok sorry

**Comment by iamluc** [comment]
Thank you @felipemarques !

**Pull request merged**